java/src/main/java/com/aliyun/odps/rodps/ROdps.java

```diff
@@ -739,4 +739,119 @@ public boolean setLogPath(String log_path) throws IOException {
     }
     return true;
   }
+
+  /***
+   * Enable MQAC
+   * @param sql
+   * @param filename
+   * @return
+   * @throws ROdpsException
+   */
+  public LinkedHashMap<String, String> runSqlWithMCQA(String sql, String filename) throws ROdpsException {
+    // If the client forget to end with a semi-colon, append it.
+    if (!sql.contains(";")) {
```

Comment:
A tailed semi-colon should be checked, because there might be other semi-colons in the middle of a legal SQL query, say "set odps.some.param=paramValue; select * from table_a limit 10"